src/strings-in-block/class-base.php

```diff
@@ -27,9 +27,16 @@ protected function get_block_config( \WP_Block_Parser_Block $block, $type ) {
 
 		if ( isset( $block->blockName, $this->block_types[ $block->blockName ][ $type ] ) ) {
 			return $this->block_types[ $block->blockName ][ $type ];
-		} else {
-			return null;
+		} elseif ( isset( $block->blockName) ) {
```

Comment:
@strategio you don't need the "else" here:

```php
protected function get_block_config( \WP_Block_Parser_Block $block, $type ) {
	if ( null === $this->block_types ) {
		$this->block_types = $this->config_option->get();
	}
	if ( isset( $block->blockName, $this->block_types[ $block->blockName ][ $type ] ) ) {
		return $this->block_types[ $block->blockName ][ $type ];
	} 
	
	if ( isset( $block->blockName) ) {
		$block_name_arr  = explode( '/', $block->blockName );
		$block_namespace = reset( $block_name_arr );
		if ( isset( $this->block_types[ $block_namespace ][ $type ] ) ) {
			return $this->block_types[ $block_namespace ][ $type ];
		}
	}
	return null;
}
```

Reply:
Right, done!